internal/platform/web/validation.go

```diff
@@ -32,6 +32,12 @@ func (err InvalidError) Error() string {
 	return str
 }
 
+// JSONError is the response for errors that occur within the API.
```

Comment:
I think moving it to `validation.go` makes it seem like JSONErrors are all because of validation. Is that true?

Reply:
They are because of the `Fields InvalidError` field. I struggled with this but that field sealed the deal for me. Probably the name of the type needs to change.

Reply:
You are right on this one. I am moving it back